dev-python/webpy/webpy-0.40.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2017 Gentoo Foundation
```

Comment:
Old, you should use repoman to commit so it fixes this for you automatically.

Reply:
Can't use repoman for now because of broken dependencies